docs/gettingstarted.rst

```diff
@@ -0,0 +1,91 @@
+Getting started with ilastik API and OMERO
+==========================================
+
+**Description**
+---------------
+
+We will use a Python script showing how to analyze data stored in an OMERO server
+using the ilastik API. The code snippets are extracted from the Python script
+:download:`pixels_classification.py <../scripts/pixels_classification.py>`. A notebook
```

Comment:
Could we maybe rename the script to say ``pixel_classification.py`` (remove the ``s``) ?